### ISSUE TITLE
How to record ranges and uncertainty for lithostratigraphic units?

### ISSUE THREAD

**Issue by karilint** [tdwg/paleo#39]
How should I present fossil data which has uncertain stratigraphical members like Moiti (the lower one) and Lokochot (the upper one)? A similar situation may occur when a fossil is found between two different stratigraphical layers like beds or volcanic tuffs.

Darwin core has a Term dwc:member but it seems to be for one name only as does abcd-efg:HostRockStratigraphy-Member too. I have also seen such terms as MeasurementOrFactAtomised/LowerValue and MeasurementOrFactAtomised/UpperValue.
Should I use a combination of those and how should I use them?

**Comment by hollyel** [comment]
I'm trying to gather some feedback to help with your question and am hoping other might chime in here. I'm wondering if [dwc:lithostratigraphicTerms](http://rs.tdwg.org/dwc/terms/lithostratigraphicTerms) could be used? Or could it be ok to have multiple values in Member that are pipe separated, so "Moiti member | Lokochot member"? That does not really help when you are trying to indicate a fossil between layers as you mentioned though. 

I hadn't thought about using MeasurementOrFact in this way. Perhaps we can get more input from others that are more familiar with ABCD EFG. I haven't really worked with those terms much. 

There are also issues noted for other sub-formation information that does not currently fit in DwC, informal unit names, and further description of the lithostratigraphy. I'm actually going to rename this Issue and create a related one for some of these other challenges with lithostratigraphy.

**Comment by RogerBurkhalter** [comment]
Because you are referring to a Member, not a Formation, I would not add the member and leave the highest rank as Formation, in this case the Koobi Fora Formation. Since these are dealing with Neogene sediments and volcanic with tight age constraints, that doesn't help researchers a whole lot and you can use dwc:earliestAgeOrLowestStage and dwc:latestAgeOrHighestStage to further refine the stratigraphic placement.

**Comment by hollyel** [comment]
Multiple people in the Paleo Data Working Group (not TDWG) have noted that their collections management systems enable recording ranges in lithostratigraphic units and some have ability to indicate uncertainty. Sounds like there are many examples of this type of data being recorded in collections that would also benefit from possible solutions found in this discussion.

**Comment by hollyel** [comment]
Adding a note to clarify that my suggestion for considering dwc:lithostratigraphicTerms here does not align to the current examples given for that term

**Comment by tucotuco** [comment]
Would you say the examples are wrong? Or misleading? Incomplete?  Examples
are not normative, so can be changed without a lot of process.

**Comment by hollyel** [comment]
Hi @tucotuco I was actually thinking about trying to pull you into this conversation to see if you had any thoughts! It seems that the current example for dwc:lithostratigraphicTerms is actually more in reference to chronostratigraphy. Therefore, it would not be a place to put lithostratigraphic units or more descriptive information about the lithology. Can you share any further information about the intended purpose of this term when it was created? And while you're here, how one might represent multiple values or ranges for a member? More convos related to these needs/confusion are in #23 and #40. THANK YOU!

**Comment by falkogloeckler** [comment]
> .... I have also seen such terms as MeasurementOrFactAtomised/LowerValue and MeasurementOrFactAtomised/UpperValue.
> Should I use a combination of those and how should I use them?

Yes! That's exactly what the StratigraphicMeasurementsAndFacts are for, too! (http://www.geocase.eu/sites/default/documentation/html/efg.html#element_StratigraphicMeasurementOrFact_Link0304E478)
It adopts the complex type from ABCD, which is defined here:  https://terms.tdwg.org/wiki/abcd2:Unit-MeasurementsOrFacts
And there it reads, that the measurement could also be categorial. Thus, you would use the atomised type (https://terms.tdwg.org/wiki/abcd2:Unit-MeasurementOrFactAtomised) to accommodate the stratigraphical range. The element "Parameter" would get the value "stratigraphical member" and the elements "lowerValue" and "upperValue" would describe the uncertain range. 

See also my comment in related issue #40
